### PR TITLE
Remove travis limit on ipopt.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
Weird, this seemed to work on the cyipopt feedstock.

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ before_install:
   - conda config --add channels conda-forge
   - conda update -q conda
 install:
-  - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION numpy cython "ipopt<3.13" future six setuptools sphinx numpydoc pkg-config
+  - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION numpy cython ipopt future six setuptools sphinx numpydoc pkg-config
   - source activate test-env
```
Comment (apommel) [inline .travis.yml:22]:
```suggestion
  - conda activate test-env
```
We may want to give it a try as source activate seems to be considered outdated.

**Comment by apommel** [comment]
Also it would be worth trying to replace
```
- export PATH=/home/travis/miniconda/bin:$PATH
```
by
```
- source "$HOME/miniconda/etc/profile.d/conda.sh"
```
It seems that Anaconda does it this way in their [documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html).

**Comment by apommel** [comment]
Any news about trying those fixes?

**Comment by moorepants** [comment]
No, just forgot about them :)

**Comment by apommel** [comment]
Travis still fails. I can't explain why on the conda-forge feedstock the install works but not here.

**Comment by moorepants** [comment]
It didn't look like it had the up-to-date travis.yml in the latest build. It still had "export ...".

**Comment by moorepants** [comment]
That got further, now getting:

```
$ python setup.py install

Package lapack was not found in the pkg-config search path.

Perhaps you should add the directory containing `lapack.pc'

to the PKG_CONFIG_PATH environment variable

Package 'lapack', required by 'ipopt', not found

Traceback (most recent call last):

  File "setup.py", line 102, in <module>

    **pkgconfig('ipopt'))]

  File "setup.py", line 58, in pkgconfig

    raise OSError(msg.format(list(packages)))

OSError: pkg-config was not able to find any of the requested packages ['ipopt'] on your system. Make sure pkg-config can discover the .pc files associated with the installed packages.

The command "python setup.py install" exited with 1.

0.02s$ python -c "import ipopt"

Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "/home/travis/build/matthias-k/cyipopt/ipopt/__init__.py", line 44, in <module>

    from cyipopt import *

ModuleNotFoundError: No module named 'cyipopt'

The command "python -c "import ipopt"" exited with 1.

0.11s$ python test/examplehs071.py

Traceback (most recent call last):

  File "test/examplehs071.py", line 26, in <module>

    import ipopt

ModuleNotFoundError: No module named 'ipopt'

The command "python test/examplehs071.py" exited with 1.

0.10s$ python test/exception_handling.py

Traceback (most recent call last):

  File "test/exception_handling.py", line 24, in <module>

    import ipopt

ModuleNotFoundError: No module named 'ipopt'

The command "python test/exception_handling.py" exited with 1.

0.11s$ python test/lasso.py

Traceback (most recent call last):

  File "test/lasso.py", line 33, in <module>

    import ipopt

ModuleNotFoundError: No module named 'ipopt'

The command "python test/lasso.py" exited with 1.

6.84s$ conda install scipy

Collecting package metadata (current_repodata.json): \ | | done

Solving environment: - done
```

**Comment by moorepants** [comment]
Now travis doesnt' seem to be running.

**Comment by moorepants** [comment]
Travis is back!

**Pull request merged**